era5_land_forcing_data2008/create_artifact.jl

```diff
@@ -0,0 +1,50 @@
+using ClimaArtifactsHelper
+using NCDatasets
+
+include("thin_and_clean_artifact.jl")
+include("find_correct_order.jl")
+
+# Get all the files that end with .nc and sort them with respect to time
+file_paths = readdir()
+nc_paths = filter(file_path -> endswith(file_path, ".nc"), file_paths) |> sort
+nc_paths = find_correct_order(nc_paths)
```

Comment:
Can you add a check that there's the correct number of NetCDF files and instruct the user to run the python script if the files are not there?